veangle-guides/staking-migration.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Staking migration guide
+---
+
+# Staking migration guide
+
+Close to the veANGLE upgrade, funds staked on Angle Ethereum mainnet staking contracts will have to be migrated. This include:
+
+- All sanTokens staking contracts
+- Gelato Uniswap agEUR/USDC LP
+- Gelato Uniswap agEUR/wETH LP
+- SushiSwap agEUR/ANG LP
+- agEUR staking if it is not discontinued
+
+Rewards will stop flowing to the older version of these contracts on the **Wednesday following the upgrade** (January 19th).
```

Comment:
Here might just be clearer to say that rewards will stop flowing on Wednesday January 19th directly

Here it's not clear what Wednesday following the upgrade (January 19th) is: could be the January 19th or January 26th